### ISSUE TITLE
Possible to use this codebase for an OpenSource Loupedeck plugin?

### ISSUE THREAD

**Issue by oddbear** [kylergib/WaveLinkPluginTouchPortal#21]
Hi, can I use this codebase for a OpenSource Loupedeck plugin?

Since there is no license file, I would like to ask, or I would need to do the research to implement this myself, some time in the future.

I have one today, but it used a library for the outdated version of WaveLink (up to version 1.5).
https://github.com/oddbear/Loupedeck.WaveLink.Plugin

**Comment by kylergib** [comment]
yes, you can use this to help. I will add a license now. would appreciate a shoutout in the repo, but is not necessary. hope this helps! sorry if the code is too messy lol
